src/screens/home/HomeScreen.tsx

```diff
@@ -45,17 +39,32 @@ export const HomeScreen = ({
         navigation={navigation}
         route={route}
         setBarStyle={setBarStyle}
-        setLauncherContext={setLauncherContext}
+        setLauncherContext={(candidateContext: LauncherContext): void => {
+          if (launcherContext.value && candidateContext.value) {
```

Comment:
Is it possible to have `candidateContext.value` being falsy?

Reply:
It should be but typescript thinks it is (because we made it that way). Have to find a better way to type infer